app/controllers/admin/users_controller.rb

```diff
@@ -21,5 +22,72 @@ def unlock_account
       redirect_to user_search_path, notice: t("admins.user_search.account_unlocking_scheduled", name: u.username)
     end
 
+    def make_admin
+      unless Role.is_admin? @user.person
```

Comment:
Style/UnlessElse: Do not use `unless` with `else`. Rewrite these with the positive case first.